docs/sre-guide/support/simple-python-package.md

```diff
@@ -0,0 +1,140 @@
+# Add a simple python package to an image we maintain
+
+This runbook describes the steps folks can take when they receive a request for
+adding a python package to an image we maintain for a community. Most requests are for
+a simple, uncomplicated package addition. This guide helps you determine if the request
+is simple, and if so, complete it.
+
+## Pre-requisites
+
+1. We (2i2c) are responsible for maintaining this image. There is currently no single source of
+   truth for determining this, unfortunately - please ask in the `#partnerships` channel if you
+   are not sure.
+
+   ```{Note}
+   If we do not maintain the image for a community, we should have a template response to be
+   sent back here.
+   ```
+
+2. The image is maintained on a GitHub repo we have full rights on.
+
+3. The image is built and pushed with [repo2docker-action](https://github.com/jupyterhub/repo2docker-action)
+
+4. The request is for a python package.
+
+5. The image is constructed in one of the following ways:
+    a. It is using repo2docker files, and has an `environment.yml` file
+    b. It is inheriting from one of the following community maintained upstream images via a `Dockerfile`.
+        i. [jupyter/docker-stacks](https://github.com/jupyter/docker-stacks)
+        ii. [pangeo-docker-images](https://github.com/pangeo-data/pangeo-docker-images/)
+
+If *any* of these pre-requisites are not met, escalate to all of engineering to figure this out.
+
+## Determine if this is a 'simple' package addition
+
+### Q1: Is the python package being requested available on `pip` or the `conda-forge` channel?
+
+- [conda-forge search](https://anaconda.org/search). Verify that the package is in the `conda-forge` channel here, rather than in any other channel.
+- [pypi search](https://pypi.org/)
+
+#### No
+
+- The package is from GitHub -> Tell the requester to release it on PyPI, and we
+  can install it from there. In the meantime, they can test it within their
+  environment by just `pip install`ing from their github repo.
+
+- Package is available in a non-conda-forge channel -> Escalate to rest of the
+  team, as mixing conda channels can get messy and complex.
+
+#### Yes
+
+Go to Q2.
+
+### Q2: Is this package part of the ML ecosystem?
+
+There are two distinct ML ecosystems in python - based on
+[tensorflow](https://www.tensorflow.org/) and [pytorch](https://pytorch.org/).
+Does the package depend transitively on either of these packages?
+
+```{note}
+We need to provide a simple way to determine this transitive dependency. Currently
+it's up to the implementor to figure that out
```

Comment:
@GeorgianaElena I would think it is whoever is following this guide. However, I actually just did some research and wrote down *how* you would do this task instead of leaving it up to unwritten knowledge :)